tests/BOAClient.test.ts

```diff
@@ -815,35 +816,138 @@ describe('BOA Client', () => {
                     {
                         "utxo": "0x3451d94322524e3923fd26f0597fb8a9cdbf3a9427c38ed1ca61104796d39c5b9b5ea33d576f17c2dc17bebc5d84a0559de8c8c521dfe725d4c352255fc71e85",
                         "unlock": {
-                            "bytes": "cZxcDw1Njiwv2w4fGc552PlRVID9Eg4al7HST0+QeS3vAO/ieiAC206EAHLokgzywh6TFbpjj7DF/90XHyOaAg=="
+                            "bytes": "ODMPbmbTEP1rLeAjCG1HBubfsWF2m7GzFpbUKuKJZDq+z5tb5m2QVkxUrEpihmUI8axkJUc11w6kaP34o4QqBg=="
                         },
                         "unlock_age": 0
                     },
                     {
                         "utxo": "0xfca92fe76629311c6208a49e89cb26f5260777278cd8b272e7bb3021adf429957fd6844eb3b8ff64a1f6074126163fd636877fa92a1f4329c5116873161fbaf8",
                         "unlock": {
-                            "bytes": "cZxcDw1Njiwv2w4fGc552PlRVID9Eg4al7HST0+QeS3vAO/ieiAC206EAHLokgzywh6TFbpjj7DF/90XHyOaAg=="
+                            "bytes": "ODMPbmbTEP1rLeAjCG1HBubfsWF2m7GzFpbUKuKJZDq+z5tb5m2QVkxUrEpihmUI8axkJUc11w6kaP34o4QqBg=="
                         },
                         "unlock_age": 0
                     },
                     {
                         "utxo": "0x7e1958dbe6839d8520d65013bbc85d36d47a9f64cf608cc66c0d816f0b45f5c8a85a8990725ffbb1ab13c3c65b45fdc06f4745d455e00e1068c4c5c0b661d685",
                         "unlock": {
-                            "bytes": "cZxcDw1Njiwv2w4fGc552PlRVID9Eg4al7HST0+QeS3vAO/ieiAC206EAHLokgzywh6TFbpjj7DF/90XHyOaAg=="
+                            "bytes": "ODMPbmbTEP1rLeAjCG1HBubfsWF2m7GzFpbUKuKJZDq+z5tb5m2QVkxUrEpihmUI8axkJUc11w6kaP34o4QqBg=="
                         },
                         "unlock_age": 0
                     }
                 ],
                 "outputs": [
                     {
-                        "value": "500000",
+                        "value": "100000",
                         "lock": {
                             "type": 0,
-                            "bytes": "nMY5oTUvd/IlFgxC/4kaavpbRqEaaalJIIbXeAZ29Co="
+                            "bytes": "2L1pan7b6W/iwgJYgbeuPqYliGn7UoBmGVZkWa6O8U8="
                         }
+                    }
+                ],
+                "payload": "0x617461642065746f76",
+                "lock_height": "0"
+            };
+
+        let tx = builder
+            .assignPayload(vote_data)
+            .sign(boasdk.TxType.Payment, tx_fee, payload_fee);
+
+        tx.inputs.forEach((value, idx) => {
+            expected.inputs[idx].unlock = value.unlock.toJSON();
+        });
+
+        assert.strictEqual(JSON.stringify(expected), JSON.stringify(tx));
```

Comment:
```suggestion
        assert.strictEqual(JSON.stringify(tx), JSON.stringify(expected));
```
The actual value is the first and expected is the second parameter.

Reply:
Ok I fixed it